spec/benchmark-spec.js

```diff
@@ -0,0 +1,21 @@
+/** @babel */
+/* eslint-env jasmine, atomtest */
+
+/* This file contains all specs to test the startup/load-performance indicated by THE timecop. */
+
+describe('editorconfig', () => {
+	beforeEach(() => {
+		waitsForPromise(() => atom.packages.activatePackage('editorconfig'));
+	});
+
+	describe('when activated', () => {
+		const loadLimit = 50; // sets the limit for package-loading in ms.
+		it(`should being loaded in less than ${loadLimit}ms`, () => {
+			expect(atom.packages.getLoadedPackage('editorconfig').loadTime).toBeLessThan(loadLimit);
```

Comment:
I would argue that a performance improvement wouldn't need a spec to go along with it. I understand that value of a spec per PR, but these tests end up causing more headaches usually.